Lib/test/test_plistlib.py

```diff
@@ -86,6 +86,19 @@
 </plist>
 """.replace(" " * 8, "\t")  # Apple as well as plistlib.py output hard tabs
 
+XML_PLIST_WITH_ENTITY = b'''\
```

Comment:
If you keep this ugly (as original), it's easier to compare with the backported commit and hence easier to review.


```suggestion
XML_PLIST_WITH_ENTITY=b'''\
```

Reply:
Improvement reverted.